client/src/remote/Plugins.js

```diff
@@ -75,6 +85,20 @@ export default class Plugins {
       return PLUGINS_PROTOCOL;
     };
 
+    // properties panel
```

Comment:
Consider mapping the global export to the mental "require" model defined in `camunda-modeler-plugin-helpers`:

We do import from `vendor` here:

```
import { useState } from "camunda-modeler-plugin-helper/vendor/@bpmn-io/properties-panel/preact/hooks";
```

Reply:
Hmm so this would mean we assign the propertiesPanel to `global.vendor`, right? We can try this out.

Reply:
I've implemented this and created a follow-up issue for the plugin helpers: https://github.com/camunda/camunda-modeler-plugin-helpers/issues/17